spdx/parsers/jsonyamlxml.py

```diff
@@ -885,20 +885,27 @@ def parse_file_attribution_text(self, file_attribution_texts):
         Parse File attribution texts
         - file_attribution_texts: list in yaml, json and string in xml format
         """
-        if isinstance(file_attribution_texts, list) or isinstance(
-            file_attribution_texts, str
-        ):
+        if isinstance(file_attribution_texts, list):
             for file_attribution_text in file_attribution_texts:
                 try:
                     return self.builder.set_file_attribution_text(
-                        self.document, file_attribution_texts
+                        self.document, file_attribution_text
                     )
                 except CardinalityError:
                     self.more_than_one_error("FILE_ATTRIBUTION_TEXT")
                 except OrderError:
                     self.order_error("FILE_ATTRIBUTION_TEXT", "FILE_NAME")
             else:
                 self.value_error("FILE_ATTRIBUTION_TEXT", file_attribution_texts)
+        elif isinstance(file_attribution_texts, str):
+            try:
+                return self.builder.set_file_attribution_text(
+                    self.document, file_attribution_texts
+                )
+            except CardinalityError:
+                self.more_than_one_error("FILE_ATTRIBUTION_TEXT")
+            except OrderError:
+                self.order_error("FILE_ATTRIBUTION_TEXT", "FILE_NAME")
```

Comment:
I couldn't find a spot where these errors are raised. If you can't find one either, we can just remove the try/catch blocks

Reply:
I think we will leave it to the larger refactoring